b2sdk/file_version.py

```diff
@@ -58,19 +59,19 @@ class BaseFileVersion:
     }
 
     def __init__(
-        self,
-        api: 'B2Api',
-        id_: str,
-        file_name: str,
-        size: int,
-        content_type: Optional[str],
-        content_sha1: Optional[str],
-        file_info: Dict[str, str],
-        upload_timestamp: int,
-        server_side_encryption: EncryptionSetting,
-        file_retention: FileRetentionSetting = NO_RETENTION_FILE_SETTING,
-        legal_hold: LegalHold = LegalHold.UNSET,
-        replication_status: Optional[ReplicationStatus] = None,
+            self,
+            api: 'B2Api',
+            id_: str,
+            file_name: str,
+            size: int,
+            content_type: Optional[str],
+            content_sha1: Optional[str],
+            file_info: Dict[str, str],
+            upload_timestamp: int,
+            server_side_encryption: EncryptionSetting,
+            file_retention: FileRetentionSetting = NO_RETENTION_FILE_SETTING,
+            legal_hold: LegalHold = LegalHold.UNSET,
+            replication_status: Optional[ReplicationStatus] = None,
```

Comment:
our linter will not like that. Please don't reformat the code that is not related to the PR

Reply:
My apologies. I did not notice this change. It may have occurred when running `nox -s format`

Reply:
The invalid linting change has been reverted